nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -95,33 +95,31 @@ def forward(self, x, att_mask=None, pos_emb=None, pad_mask=None):
         Returns:
             x (torch.Tensor): (B, T, d_model)
         """
+        dtype = x.dtype
         residual = x
         x = self.norm_feed_forward1(x)
         x = self.feed_forward1(x)
-        x = self.fc_factor * self.dropout(x) + residual
+        residual = residual + self.dropout(x) * self.fc_factor
```

Comment:
Was there any specific reason naming of x and residual are changed here?

Reply:
With ONNX opset 15, I have encountered some errors related to aliasing. Those changes fixed that.